### DISCUSSION TITLE
Beautiful Table of Contents with navigation

### DISCUSSION THREAD

**Discussion by ManasMadrecha** [nuxt/content#1075] [General]
Loved this new UI of showing previous and next page in the TOC itself. ❤️ 

![image](https://user-images.githubusercontent.com/65993481/168133414-8aaa36fa-74ac-434f-a470-5c3c7fcc93c8.png)

Very useful as well, because it enables quick navigation, instead of scrolling all the way down and then clicking the next post. 

Kudos to whoever designed it. 👍🏻😃

**Comment by leosin** [comment]
cool 👍
